src/test/scala/org/tessellation/consensus/CellMonoidTest.scala

```diff
@@ -0,0 +1,132 @@
+
+  implicit val M: Monoid[Cell[IO, StackF, Ω, Either[CellError, Ω], Ω]] = Cell.cellMonoid[IO, StackF]
+
+  "empty" - {
+    "creates empty" in {
+      val empty = M.empty
+      empty.data.isInstanceOf[NullTerminal] shouldBe true
+    }
+
+    "empty hylo is just an identity" in {
+      val empty = M.empty
+      val hylo = empty.run()
+      val input = empty.data
+      val output = hylo.unsafeRunSync()
+      output shouldBe Right(input)
+    }
+
+    "combine is an identity function (right empty)" in {
+      val metadata = L1ConsensusMetadata.empty(
+        L1ConsensusContext(
+          Node("A", RandomTransactionGenerator("A")),
+          Set.empty[Node],
+          RandomTransactionGenerator("A")
+        )
+      )
+      val l1Cell = L1StartConsensusCell(L1Edge(Set.empty[L1Transaction]), metadata)
+      val emptyCell = M.empty
+      val combined = M.combine(
+        l1Cell,
+        emptyCell
+      )
+
+      combined shouldBe l1Cell
+    }
+
+    "combine is an identity function (left empty)" in {
+      val metadata = L1ConsensusMetadata.empty(
+        L1ConsensusContext(
+          Node("A", RandomTransactionGenerator("A")),
+          Set.empty[Node],
+          RandomTransactionGenerator("A")
+        )
+      )
+      val l1Cell = L1StartConsensusCell(L1Edge(Set.empty[L1Transaction]), metadata)
+      val emptyCell = M.empty
+      val combined = M.combine(
+        emptyCell,
+        l1Cell
+      )
+
+      combined shouldBe l1Cell
+    }
+
+    "combine same types" in {
+      val metadata = L1ConsensusMetadata.empty(
+        L1ConsensusContext(
+          Node("A", RandomTransactionGenerator("A")),
+          Set.empty[Node],
+          RandomTransactionGenerator("A")
+        )
+      )
+      val txA = L1Transaction(1, "a", "b", "", 0)
+      val l1CellA = L1StartConsensusCell(L1Edge(Set(txA)), metadata)
+      val txB = L1Transaction(1, "b", "c", "", 0)
+      val l1CellB = L1StartConsensusCell(L1Edge(Set(txB)), metadata)
+
+      val resultA = l1CellA.run().unsafeRunSync()
+      val resultB = l1CellB.run().unsafeRunSync()
+      val expectedResult = resultA.flatMap(a => resultB.map(b => a :: b))
+
+      val combined = M.combine(
+        l1CellA,
+        l1CellB
+      )
+
+      val result = combined.run().unsafeRunSync()
+
+      result shouldBe expectedResult
+    }
+
+    "combine different types" in {
+      val metadata = L1ConsensusMetadata.empty(
+        L1ConsensusContext(
+          Node("A", RandomTransactionGenerator("A")),
+          Set.empty[Node],
+          RandomTransactionGenerator("A")
+        )
+      )
+      val l1Cell = L1StartConsensusCell(L1Edge(Set.empty[L1Transaction]), metadata)
+      val l0Cell = L0Cell(L0Edge(Set.empty[L1Block]))
+
+      val l1Result = l1Cell.run().unsafeRunSync()
+      val l0Result = l0Cell.run().unsafeRunSync()
+      val expectedResult = l1Result.flatMap(a => l0Result.map(b => a :: b))
+
+      // TODO: Investigate why syntax doesn't work
```

Comment:
The import should be `cats.syntax.all._` instead of `cats.syntax._`.

Reply:
Didn't help, unfortunately.